packages/target-electron/src/windows/video-call.ts

```diff
@@ -572,10 +632,72 @@ function isOriginGood(url: string) {
   }
   return (
     urlParsed.protocol === SCHEME_NAME + ':' &&
-    urlParsed.host === DUMMY_HOST_NAME &&
+    parseHost(urlParsed.host) != null &&
     urlParsed.port === ''
   )
 }
+function formatHost(
+  accountId: number,
+  chatIdOrMessageId:
+    | {
+        chatId: number
+        callMessageId?: undefined
+      }
+    | {
+        chatId?: undefined
+        callMessageId: number
+      }
+): string {
+  return (
+    `${chatIdOrMessageId.callMessageId ?? 'none'}.` +
+    `${chatIdOrMessageId.chatId ?? 'none'}.` +
+    `${accountId}.` +
+    `calls-webapp-dummy-host`
+  )
+}
+function parseHost(host: string): null | {
```

Comment:
```suggestion
/**
 * check if the numbers used in host are valid
 * and the format (see formatHost) is correct
 * @returns `null` if the host is invalid.
 */
function parseHost(host: string): null | {
```

Reply:
```suggestion
/**
 * @see {@linkcode formatHost}.
 * @returns `null` if the host is invalid.
 */
function parseHost(host: string): null | {
```

The first 2 lines are redundant IMO. This info can be inferred from the return type.